docker-compose.yml

```diff
@@ -78,8 +78,10 @@ services:
       - OTEL_EXPORTER_OTLP_TRACES_ENDPOINT
       - OTEL_EXPORTER_OTLP_METRICS_ENDPOINT
       - OTEL_EXPORTER_OTLP_METRICS_TEMPORALITY_PREFERENCE
+      - OTEL_EXPORTER_OTLP_LOGS_ENDPOINT
       - OTEL_RESOURCE_ATTRIBUTES
       - OTEL_SERVICE_NAME=adservice
+      - OTEL_LOGS_EXPORTER=otlp
```

Comment:
Just a nit picky, maybe for better maintainability. 
Could we move `OTEL_LOGS_EXPORTER=otlp` to `.env` and leave it in here just the `OTEL_LOGS_EXPORTER`?